modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
 				'type' => Controls_Manager::DIMENSIONS,
```

Comment:
The 'content_padding' control should be updated.
I believe the original screenshots were correct, and should be reverted.

Reply:
Reverted